.github/workflows/main.yml

```diff
@@ -15,91 +15,102 @@ on:
       - 'v*.*.*'
 
 env:
-  # Change this to invalidate existing cache.
-  CACHE_PREFIX: v3
+  CACHE_PREFIX: v3  # Change this to invalidate existing cache.
   PYTHON_PATH: ./
   DEFAULT_PYTHON: 3.7
+  BEAKER_TOKEN: ${{ secrets.BEAKER_TOKEN }}
+  BEAKER_WORKSPACE: ai2/tango-testing
+  BEAKER_CLUSTER: ai2/tango-gpu-tests
+  BEAKER_IMAGE: petew/tango-testing
 
 jobs:
   checks:
-    name: python ${{ matrix.python }} - ${{ matrix.task.name }}
+    name: ${{ matrix.task.name }}
     runs-on: [ubuntu-latest]
     timeout-minutes: 30
     strategy:
       fail-fast: false
       matrix:
         python: ['3.7']
         task:
-          - name: Lint
+          - name: python 3.7 - Lint
```

Comment:
Isn't it bad to hard-code the version number like this?

Reply:
It's not ideal, but the GPU tests shouldn't have the Python version in their name. That's the one exception, which is why I had to change this.

Why shouldn't the GPU tests have the Python version in their name? Because the Python version here is not relevant to the GPU tests since the GPU tests are ran in a Docker container which may be using a different Python version.